<component name="InspectionProjectProfileManager">
  <profile version="1.0">
    <option name="myName" value="Project Default" />
    <inspection_tool class="BadExceptionCaught" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true">
        <option name="exceptionsString" value="" />
        <option name="exceptions">
          <value />
        </option>
      </scope>
      <option name="exceptionsString" value="" />
      <option name="exceptions">
        <value />
      </option>
    </inspection_tool>
    <inspection_tool class="BadExceptionDeclared" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true">
        <option name="exceptionsString" value="" />
        <option name="exceptions">
          <value />
        </option>
        <option name="ignoreTestCases" value="false" />
        <option name="ignoreLibraryOverrides" value="false" />
      </scope>
      <option name="exceptionsString" value="" />
      <option name="exceptions">
        <value />
      </option>
      <option name="ignoreTestCases" value="false" />
      <option name="ignoreLibraryOverrides" value="false" />
    </inspection_tool>
    <inspection_tool class="BadExceptionThrown" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true">
        <option name="exceptionsString" value="" />
        <option name="exceptions">
          <value />
        </option>
      </scope>
      <option name="exceptionsString" value="" />
      <option name="exceptions">
        <value />
      </option>
    </inspection_tool>
    <inspection_tool class="CaughtExceptionImmediatelyRethrown" enabled="true" level="ERROR" enabled_by_default="true">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="CheckedExceptionClass" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="ContinueOrBreakFromFinallyBlock" enabled="true" level="ERROR" enabled_by_default="true">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="EmptyCatchBlock" enabled="true" level="ERROR" enabled_by_default="true">
      <scope name="Problems" level="ERROR" enabled="true">
        <option name="m_includeComments" value="true" />
        <option name="m_ignoreTestCases" value="true" />
        <option name="m_ignoreIgnoreParameter" value="true" />
      </scope>
      <option name="m_includeComments" value="true" />
      <option name="m_ignoreTestCases" value="true" />
      <option name="m_ignoreIgnoreParameter" value="true" />
    </inspection_tool>
    <inspection_tool class="EmptyFinallyBlock" enabled="true" level="ERROR" enabled_by_default="true">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="EmptyTryBlock" enabled="true" level="ERROR" enabled_by_default="true">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="ErrorRethrown" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="ExceptionFromCatchWhichDoesntWrap" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true">
        <option name="ignoreGetMessage" value="false" />
        <option name="ignoreCantWrap" value="false" />
      </scope>
      <option name="ignoreGetMessage" value="false" />
      <option name="ignoreCantWrap" value="false" />
    </inspection_tool>
    <inspection_tool class="ExtendsThrowable" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="FinallyBlockCannotCompleteNormally" enabled="true" level="ERROR" enabled_by_default="true">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="InstanceofCatchParameter" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="LoggerInitializedWithForeignClass" enabled="false" level="WARNING" enabled_by_default="false">
      <option name="loggerClassName" value="org.apache.log4j.Logger,org.slf4j.LoggerFactory,org.apache.commons.logging.LogFactory,java.util.logging.Logger" />
      <option name="loggerFactoryMethodName" value="getLogger,getLogger,getLog,getLogger" />
    </inspection_tool>
    <inspection_tool class="NestedTryStatement" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="NewExceptionWithoutArguments" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="NonFinalFieldOfException" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="NullThrown" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="ReturnFromFinallyBlock" enabled="true" level="ERROR" enabled_by_default="true">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="ThreadDeathRethrown" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="ThrowCaughtLocally" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true">
        <option name="ignoreRethrownExceptions" value="false" />
      </scope>
      <option name="ignoreRethrownExceptions" value="false" />
    </inspection_tool>
    <inspection_tool class="ThrowFromFinallyBlock" enabled="true" level="ERROR" enabled_by_default="true">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="ThrowsRuntimeException" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="TooBroadCatch" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="TooBroadThrows" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="UncheckedExceptionClass" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="UnnecessaryInitCause" enabled="true" level="ERROR" enabled_by_default="true">
      <scope name="Problems" level="ERROR" enabled="true" />
    </inspection_tool>
    <inspection_tool class="UnusedCatchParameter" enabled="true" level="ERROR" enabled_by_default="false">
      <scope name="Problems" level="ERROR" enabled="true">
        <option name="m_ignoreCatchBlocksWithComments" value="false" />
        <option name="m_ignoreTestCases" value="false" />
      </scope>
      <option name="m_ignoreCatchBlocksWithComments" value="false" />
      <option name="m_ignoreTestCases" value="false" />
    </inspection_tool>
  </profile>
</component>